powerVCF.psm1

```diff
@@ -1877,16 +1874,301 @@ Function Set-VCFCeip {
 		    $response = Invoke-RestMethod -Method PATCH -URI $uri -ContentType application/json -headers $headers -body $ConfigJson
 		    $response      
         }
-        catch {
-            # Call the function ResponseExeception which handles execption messages
-            ResponseExeception
-        }
-
+    catch {
+        # Call the function ResponseExeception which handles execption messages
+        ResponseExeception
+    }
 }
 Export-ModuleMember -Function Set-VCFCeip
 
 ######### End CEIP Operations ##########
 
+######### Start Backup Configuration Operations ##########
+
+Function Get-VCFBackupConfiguration {
+<#
+    .SYNOPSIS
+    Gets the backup configuration of NSX Manager and SDDC Manager
+	
+    .DESCRIPTION
+     Retrieves the backup configuration details and the status
+	
+    .EXAMPLE
+    This example shows the backup configuration
+	
+	PS C:\> Get-VCFBackupConfiguration
+#>
+
+    $headers = @{"Accept" = "application/json"}
+    $headers.Add("Authorization", "Basic $base64AuthInfo")
+    $uri = "https://$sddcManager/v1/system/backup-configuration"
+
+    try { 	
+		    $response = Invoke-RestMethod -Method GET -URI $uri -headers $headers
+		    $response
+        }
+    catch {
+        # Call the function ResponseExeception which handles execption messages
+        ResponseExeception
+    }
+}
+Export-ModuleMember -Function Get-VCFBackupConfiguration
+
+######### End Backup Configuration Operations ##########
+
+######### Start Bundle Operations ##########
+
+Function Get-VCFBundles {
```

Comment:
It would be good to expand this in future to be able to get/search for a specific bundle. For that reason I'd probably call it Get-VCFBundle

Reply:
Agreed, removed 's'. Yes expanding will happen for sure, right now just adding basic capabilities